operator/roles/kiali-deploy/templates/openshift/console-links.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
+metadata:
+  name: kiali-app-{{ kiali_vars.deployment.namespace }}
+  labels:
+    app: kiali
+    version: {{ kiali_vars.deployment.version_label }}
+    kiali.io/home: {{ kiali_vars.deployment.namespace }}
+spec:
+  href: {{ kiali_route_url }}
+  location: ApplicationMenu
+  text: Kiali
+{% for namespace in namespaces %}
+---
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
+metadata:
+  name: kiali-namespace-{{ namespace }}
+  labels:
+    app: kiali
+    version: {{ kiali_vars.deployment.version_label }}
+    kiali.io/home: {{ kiali_vars.deployment.namespace }}
+spec:
+  href: {{ kiali_route_url }}/{{ kiali_vars.server.web_root }}/console/graph/namespaces?namespaces={{ namespace }}
```

Comment:
we should fix this - if kiali_vars.server.web_root is "/" that this will show up as "///"

Reply:
```suggestion
  href: {{ kiali_route_url }}{{ '/' if kiali_vars.server.web_root == '/' else '/'+kiali_vars.server.web_root }}/console/graph/namespaces?namespaces={{ namespace }}
```

Reply:
this is a separate issue - we'll fix after we merge this PR: https://github.com/kiali/kiali/issues/2132